.github/workflows/configure.yml

```diff
@@ -216,15 +216,15 @@ jobs:
         show-progress: 'false'
 
     - name: Add ubuntu mirrors
-      if: runner.os == 'Linux' && matrix.packages
+      if: runner.os == 'Linux' && matrix.packages && !contains(matrix.os, 'z15')
```

Comment:
**thought:** 
These are both valid ways to do this, but personally I prefer the existing way as it doesn't add special handling for specific actions runners to these tasks.

These are both skipped when `matrix.packages` is empty already.

Reply:
I know the end result is the same, but there really isn't any justification for configure workflow to do it in one way and cmake workflow to do in another way.

Reply:
> **suggestion:**
> True enough, then perhaps the cmake workflow should be changed instead.

You are free to open own PR for that but that doesn't mean everyone will agree that it will be any cleaner or better solution.